pkg/config/config.go

```diff
@@ -178,6 +178,16 @@ func (vm *VirtualMachine) VirtioVsockDevices() []*VirtioVsock {
 	return vsockDevs
 }
 
+func (vm *VirtualMachine) NetworkBlockDevice(deviceID string) *NetworkBlockDevice {
```

Comment:
Is there a limit of 1 network block device per-VM?

Reply:
No you can have more. This is used when we want to retrieve info about one specific nbd from https://github.com/crc-org/vfkit/pull/212/files#diff-04ef060002636f3776175738389d7448ca395b65d0db23da5137ade7c279a418R382

Reply:
Ah oops, did not pay attention to the `deviceId` parameter